src/test/java/SalesforceModules/Leads.java

```diff
@@ -0,0 +1,229 @@
+package SalesforceModules;
+
+import com.relevantcodes.extentreports.LogStatus;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import pageobjects.SalesforceObjects.LeadsPage;
+import step_definitions.RunCukesTest;
+import utilities.ConfigReader;
+import utilities.HTMLReportUtil;
+import utilities.KeywordUtil;
+
+import java.util.HashMap;
+
+import static org.testng.Assert.assertEquals;
+
+
+public class Leads {
+    public static HashMap<String, String> dataMap = new HashMap<String, String>();
+
+    public static void ClickOnHamburgerMenu(String logStep) throws InterruptedException {
```

Comment:
Please use consistent naming conventions for functions name, change the 'ClickOnHamburgerMenu' to 'clickOnHamburgerMenu' and 'EnterText' to 'enterText'

Reply:
updated method names